src/transformers/models/deit/modeling_deit.py

```diff
@@ -297,12 +297,6 @@ def forward(self, hidden_states, head_mask=None, output_attentions=False):
 
         # in DeiT, layernorm is also applied after self-attention
         layer_output = self.layernorm_after(hidden_states)
-
```

Comment:
Should be done in its own PR too? Unless you're relying on this with some Copied from.

Reply:
Yes it's because of Copied from statements which I added.